enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1334 @@
+progress toward instead of the "latest".
+
+```yaml
+kind: MachineConfigPool
+spec:
+  machineConfig:
+    # An administrator can identify the exact rendered MachineConfig
+    # the MCO should progress towards for this MCP.
+    name: <rendered machine configuration name>
+    
+    # Validation defines when MCO will allow the use of a new
+    # configuration. Default means a node must successfully use the
+    # configuration before new nodes are ignited with the
+    # desiredConfiguration.
+    # None means no validation is necessary (new nodes will
+    # immediately ignite with the configuration).
+    validation: Default
+```
+
+### Projected ClusterVersion in HCP
+A [Cluster Instance Admin](https://hypershift-docs.netlify.app/reference/concepts-and-personas/#personas) using a 
+hosted cluster cannot directly update their cluster. When they query `ClusterVersion` on their cluster, they only
+see a "projected" value. If they edit the projected value, it does not affect change on the cluster.
+
+In order to update the hosted cluster, changes must be made to its associated HCP resources on the management cluster.
+Since this proposal subjects those resources to change management control, this information must also be projected
+into the `ClusterVersion` (including change management status information) of hosted clusters so that a Cluster 
+Instance Admin can understand when material changes to their cluster will take place.
+
+### Workflow Description
+
+#### OCM HCP Standard Change Management Scenario
+
+1. A [Cluster Service Consumer](https://hypershift-docs.netlify.app/reference/concepts-and-personas/#personas) requests an HCP cluster via OCM.
+1. To comply with their company policy, the service consumer configures a maintenance schedule through OCM. 
+1. Their first preference, no updates at all, is rejected by OCM policy, and they are referred to service 
+   delivery documentation explaining minimum requirements. 
+1. The user specifies a policy which permits control-plane changes to be initiated any time Saturday UTC on the control-plane.
+1. To limit overall workload disruption, the user changes their worker-node policy to permit updates only on the **first** Saturday of each month.
+1. OCM accepts the configuration.
+1. OCM configures the HCP (HostedCluster/NodePool) resources via the Service Delivery Hive deployment to contain a `changeManagement` stanza 
+   referring to newly created `ChanageManagementPolicy` objects using the `MaintenanceSchedule` strategy.
+1. Hive creates/updates the associated HCP resources.
+1. Company workloads are added to the new cluster and the cluster provides business value.
+1. To leverage a new feature in OpenShift, the service consumer plans to update the minor version of the platform.
+1. Via OCM, the service consumer requests the minor version update. They can do this at any time with confidence that the maintenance 

```

Comment:
Do we feel we are at a point where our customers trust us to handle upgrades in a way that they can just walk away?